src/test/unit/components/GameCard.test.tsx

```diff
@@ -37,4 +82,14 @@ describe('Rendering Game GameCard', () => {
     const summary = screen.getByText(/Surprisingly few dragons./);
     expect(summary).toBeInTheDocument();
   })
+
+  it('renders select menu', async () => {
```

Comment:
Is it possible to also test that it displays the correct libraries?

What about when you select and submit a library to add to?

Reply:
I was having trouble figuring out how to test this to be honest. Do you know of any testing libraries that can allow a user to click components and view things? I was playing with it a bit but got a bit confused (and, truthfully, gave up semi-quickly lol)

Reply:
You can click with RTL! https://testing-library.com/docs/dom-testing-library/api-events/